examples/aks/aks_cluster_gitops/README.md

```diff
@@ -34,28 +42,32 @@ Prerequisites:
 
 
 ### Step 1 & 2: Update TF vars & TF Init, plan & apply
-After successful apply, CAST Console UI will be in `Connecting` state. \
-Note generated 'CASTAI_CLUSTER_ID' from outputs
+After successful apply, CAST Console UI will be in `Connecting` state.
 
+### Step 3: Execute TF output command & save the below output values
+terraform output cluster_id  
+terraform output cluster_token
+
+Obtained values are needed for next step
 
 ### Step 3: Deploy Helm chart of CAST Components
 Coponents: `castai-cluster-controller`,`castai-evictor`, `castai-spot-handler`, `castai-kvisor`, `castai-workload-autoscaler`, `castai-pod-pinner` \
 After all CAST AI components are installed in the cluster its status in CAST AI console would change from `Connecting` to `Connected` which means that cluster onboarding process completed successfully.
 
 ```
-CASTAI_API_KEY=""
-CASTAI_CLUSTER_ID=""
+CASTAI_CLUSTER_ID="<Replace cluster_id>"
+CASTAI_CLUSTER_API_TOKEN="<Replace cluster_token>"
```

Comment:
May be it's better to use _KEY here as well?

Reply:
https://github.com/castai/terraform-provider-castai/pull/424#discussion_r1846563344